wallet/src/de/schildbach/wallet/ui/OnboardingViewModel.kt

```diff
@@ -78,6 +78,7 @@ class OnboardingViewModel @Inject constructor(
             if (!walletApplication.isWalletUpgradedToBIP44) {
                 walletApplication.wallet!!.addKeyChain(Constants.BIP44_PATH)
             }
+            (walletApplication.wallet as WalletEx).initializeCoinJoin(0)
```

Comment:
We need to add the CoinJoin keychain to the wallet when upgrading from Dash Wallet.  In other places, we need to do the same when restoring from a backup file.